src/hooks/Guilds/useConfig.ts

```diff
@@ -0,0 +1,45 @@
+import useEtherSWR from 'ether-swr';

```

Comment:
This is good, we also discussed using a wrapping style like this for all swr calls so that we can test it easier, once we have that we can move this in there also

Reply:
this is ok @rossneilson, and enough for mocking ! 
because we can mock the useConfig hook.